### PR TITLE
Simple subscription form

### PR DESCRIPTION
**Pull request by AllyReid**
Added simple subscription form 

https://github.com/zurb/building-blocks/issues/82

### CODE REVIEW

**Comment by IamManchanda** [comment]
Would be even great if you add background to it 
and also define color and background as a SASS Variable to let user  customize easily

**Comment by AllyReid** [comment]
@IamManchanda ,

Wasn't sure how much styling to add initially so kept it pretty light. I've updated the scss with a couple more options.

**Comment by IamManchanda** [comment]
LGTM

File: src/building-blocks/simple-subscription-form/simple-subscription-form.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+$global-radius: 5px;
+$subscription-form-bg: #0d3d5d;
```
Comment (kball) [inline src/building-blocks/simple-subscription-form/simple-subscription-form.scss]: Can we tie this into a settings variable to make it automatically fit the user's theme? Could either directly be the variable, or a darkened version of it? e.g.
```
$subscription-form-bg: darken($primary-color, 20%);
```

**Comment by AllyReid** [comment]
@kball 

I have updated those but for some reason the $global-radius variable doesn't seem to work unless set within the block. 

Others like $white and the rem-calc() function are fine 🤔

**Comment by AllyReid** [comment]
@kball also removed an auto compiled css file created by Brackets 😳

**Comment by kball** [comment]
Within the building blocks environment, these are picking up the default settings values, and the default for `$global-radius` is 0... putting this override in is actually resetting global-radius, which is not ideal. I'd recommend removing the global reference setting - it works, but in the BB view it will be flat/not rounded. Other than that this looks good to go

**Comment by kball** [comment]
Great! Nice work @AllyReid.

Actually one more question - Do you have a twitter handle you'd like to add to the author variable in the yml file? We're going to be adding twitter related info & doing shoutouts

**Comment by kball** [comment]
Merged!  Thanks @AllyReid

**Pull request merged**